ui/src/profile.ts

```diff
@@ -75,3 +76,14 @@ export const fetchFollowing = (): void => {
     return reqs.filter(req => req.type !== waitingRoom.Status.Cloned);
   });
 };
+
+export const showNotificationsForFailedProjects = async (): Promise<void> => {
+  const failedProjects = await project.fetchFailed();
+  failedProjects.forEach(project => {
+    notification.error(
+      `The project ${project.metadata.name} could not be loaded`,
```

Comment:
Would it make sense to have a way of displaying the type of failure? I'm thinking the more information we give the user about the failure, then the better the reports we get when they do fail.